core/vmpu/src/vmpu.c

```diff
@@ -414,6 +434,18 @@ static void vmpu_enumerate_boxes(void)
         int index = box_order[box_id];
         UvisorBoxConfig const * box_cfgtbl = ((UvisorBoxConfig const * *) __uvisor_config.cfgtbl_ptr_start)[index];
 
+        if(box_cfgtbl->debug_box_ptr != NULL) {
+            if(++debug_box_count > MAX_DEBUG_BOX_COUNT) {
+                HALT_ERROR(SANITY_CHECK_FAILED, "More than one box is registered to the debug box!\n");
```

Comment:
This won't be possible when we use uVisor config, since there will be multiple definition errors at build time if multiple debug boxes are created.